Jenkinsfile-run

```diff
@@ -55,7 +55,7 @@ pipeline {
         expression { params.CREATE_RESOURCES }
       }
       steps {
-        copyDockerImages("consignment-api")
+//        copyDockerImages("consignment-api")
```

Comment:
Did you mean to leave this commented out?

Reply:
Obviously ;)